teams/teams-ps/teams/New-CsTeamsEventsPolicy.md

```diff
@@ -132,7 +147,41 @@ Aliases:
 
 Required: False
 Position: Named
-Default value: Everyone
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowedTownhallTypesForRecordingPublish
+This setting describes how IT admins can control which types of Town Hall attendees can have their recordings published.
+
+Possible values are: None, InviteOnly, EveryoneInCompanyIncludingGuests, Everyone.
+
+```yaml
+Type: String
+Parameter Sets: (All)
+Aliases:
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -AllowedWebinarTypesForRecordingPublish
+This setting describes how IT admins can control which types of webinar attendees can have their recordings published.

```

Comment:
This description is not clear. Can you follow the same style as other parameters with something like:

"This setting governs which webinar attendees can have their recordings published".